modules/programs/sioyek.nix

```diff
@@ -0,0 +1,70 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.programs.sioyek;
+
+  renderAttrs = attrs:
+    concatStringsSep "\n"
+    (mapAttrsToList (name: value: "${name} ${value}") attrs);
+in {
+  options = {
+    programs.sioyek = {
+      enable = mkEnableOption
+        "Sioyek is a PDF viewer designed for reading research papers and technical books.";
+
+      package = mkOption {
+        default = pkgs.sioyek;
+        defaultText = literalExpression "pkgs.sioyek";
+        type = types.package;
+        description = "Package providing the sioyek binary";
+      };
+
+      bindings = mkOption {
+        description = ''
+          Input configuration written to
+          <filename>$XDG_CONFIG_HOME/sioyek/keys_user.config</filename>.
+          See <link xlink:href="https://github.com/ahrm/sioyek/blob/main/pdf_viewer/keys.config"/>.
+        '';
+        type = types.attrsOf types.str;
+        default = { };
+        example = literalExpression ''
+          {
+          "move_up" = "k";
+          "move_down" = "j";
+          "move_left" = "h";
+          "move_right" = "l";
+          }
```

Comment:
```suggestion
          {
            "move_up" = "k";
            "move_down" = "j";
            "move_left" = "h";
            "move_right" = "l";
          }
```
nit: indentation

Reply:
Ok, thanks. I copied the example from the mpv module, and noticed that they did not have indentation, so perhaps someone should make a small edit there as well.